hudi-common/src/main/java/org/apache/hudi/common/table/view/RemoteHoodieTableFileSystemView.java

```diff
@@ -179,9 +176,7 @@ private <T> T executeRequest(String requestPath, Map<String, String> queryParame
 
     String url = builder.toString();
     LOG.info("Sending request : (" + url + ")");
-    // Reset url and method, to avoid repeatedly instantiating objects.
-    urlCheckedFunc.setUrlAndMethod(url, method);
-    Response response =  retryHelper != null ? retryHelper.tryWith(urlCheckedFunc).start() : urlCheckedFunc.get();
+    Response response =  retryHelper != null ? retryHelper.tryWith(() -> get(timeoutSecs, url, method)).start() : get(timeoutSecs, url, method);
     String content = response.returnContent().asString();
```

Comment:
The logic is same as before, we should figure out why this fs view client has concurrency request conflicts. The client expects to keep thread safe by itself.